### ISSUE TITLE
how can i reproduce the results on truthfulqa?

### ISSUE THREAD

**Issue by SuperChanS** [alisawuffles/proxy-tuning#7]
I notice that operating truthfulqa.sh requires "gpt_true_model_name" and "gpt_info_model_name". But it seems the original model is unavailable now.

**Comment by alisawuffles** [comment]
Yes, this part of the evaluation is unfortunately no longer reproducable due to OpenAI's deprecation of GPT-3-based models. The [allenai/open-instruct](https://github.com/allenai/open-instruct) evaluation frameowrk has switched to finetuned judge models based on Llama2 instead! Please see their evaluation script [here](https://github.com/allenai/open-instruct/blob/ea3467dbb2376e396e10f189786d04db6b2f0408/eval/truthfulqa/run_eval.py#L491-L502).
